modules/nw-ovn-kubernetes-pod-connectivity-checks.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-pod-network-connectivity-checks_{context}"]
+= OVN-Kubernetes pod network connectivity checks
+
+The connectivity check controller, in {product-title} 4.10 and later, orchestrates connection verification checks in your cluster. These include Kubernetes API, OpenShift API and individual nodes. The results for the connection tests are stored in `PodNetworkConnectivity` objects in the `openshift-network-diagnostics` namespace. Connection tests are performed every minute in parallel.
```

Comment:
would be good to link to where this is described in the Networking guide at https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/networking/verifying-connectivity-endpoint